packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -121,7 +122,10 @@ const DarkBackgroundMenu = styled.div`
   z-index: 1;
   right: 0;
   bottom: 0;
-  border-radius: ${props => props.borderRadius || '25px'};
+  border-radius: 10px;
+  position: absolute;
+  width: 100%;
+  height: 100%;
```

Comment:
make sure this looks good for the related previous implementations you did with this component 👁️